Packs/Core/Playbooks/playbook-Uncommon_creation_or_access_operation_of_sensitive_shadow_copy_by_a_high_risk_process.yml

```diff
@@ -0,0 +1,971 @@
+      '#default#':
+      - "10"
+      (powershell.exe | ntdsutil.exe | esentutl.exe | cmd.exe:
+      - "11"
+    separatecontext: false
+    conditions:
+    - label: (powershell.exe | ntdsutil.exe | esentutl.exe | cmd.exe
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              simple: alert.initiatedby
+            iscontext: true
+          right:
+            value:
+              simple: powershell.exe
+          ignorecase: true
+        - operator: isEqualString
+          left:
+            value:
+              simple: alert.initiatedby
+            iscontext: true
+          right:
+            value:
+              simple: cmd.exe
+          ignorecase: true
+        - operator: isEqualString
+          left:
+            value:
+              simple: alert.initiatedby
+            iscontext: true
+          right:
+            value:
+              simple: esentutl.exe
+          ignorecase: true
+        - operator: isEqualString
+          left:
+            value:
+              simple: alert.initiatedby
+            iscontext: true
+          right:
+            value:
+              simple: ntdsutil.exe
+          ignorecase: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -210,
+          "y": 385
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "10":
+    id: "10"
+    taskid: cc7e9e61-7560-48b8-8ca4-cab430695e9e
+    type: condition
+    task:
+      id: cc7e9e61-7560-48b8-8ca4-cab430695e9e
+      version: -1
+      name: Check actor_process_image_name VSSVC.exe & username SYSTEM
+      description: ""
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "12"
+      "yes":
+      - "11"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              simple: alert.initiatedby
+            iscontext: true
+          right:
+            value:
+              simple: VSSVC.exe
+          ignorecase: true
+      - - operator: containsString
+          left:
+            value:
+              simple: alert.username
+            iscontext: true
+          right:
+            value:
+              simple: SYSTEM
+          ignorecase: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 110,
+          "y": 555
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "11":
+    id: "11"
+    taskid: 91d336ec-f65c-49d8-8478-2db259fda538
+    type: title
+    task:
+      id: 91d336ec-f65c-49d8-8478-2db259fda538
+      version: -1
+      name: Remediation
```

Comment:
<img width="568" alt="image" src="https://github.com/user-attachments/assets/d0b636da-8e5e-4415-bba8-abc3223750b0">
the squares are not alligned, pls allign

Reply:
also, it is not clear which arrow is going where, maybe take all the steps a bit down?
<img width="697" alt="image" src="https://github.com/user-attachments/assets/77b8d940-3533-4a03-954d-9763b9459ed0">

Reply:
don't forget to update the playbook pic after changes

Reply:
this small misalignment is to make it clearer where the lines are connected to, if its perfectly aligned the lines are joined into a single line

Reply:
So make a bigger misalignment such that it is clear